src/qibolab/dummy/platform.py

```diff
@@ -3,34 +3,34 @@
 from qibolab.components import AcquireChannel, DcChannel, IqChannel
 from qibolab.instruments.dummy import DummyInstrument, DummyLocalOscillator
 from qibolab.platform import Platform
+from qibolab.qubits import Qubit
 
 FOLDER = pathlib.Path(__file__).parent
 
 
-def create_dummy():
+def create_dummy() -> Platform:
     """Create a dummy platform using the dummy instrument."""
     # register the instruments
     instrument = DummyInstrument("dummy", "0.0.0.0")
-    twpa_pump = DummyLocalOscillator("twpa_pump", "0.0.0.0")
-
-    platform = Platform.load(path=FOLDER, instruments=[instrument, twpa_pump])
+    pump = DummyLocalOscillator("twpa_pump", "0.0.0.0")
 
+    qubits = {}
     # attach the channels
-    for q, qubit in platform.qubits.items():
-        acquisition_name = f"qubit_{q}/acquire"
-        probe_name = f"qubit_{q}/probe"
-        qubit.probe = IqChannel(
-            probe_name, mixer=None, lo=None, acquisition=acquisition_name
-        )
-        qubit.acquisition = AcquireChannel(
-            acquisition_name, twpa_pump=twpa_pump.name, probe=probe_name
+    for q in range(5):
```

Comment:
In principle here we are hard-coding the qubit names (here it happens to just be integers), as I was saying in the other comment, but I believe this is a fine compromise given all the other simplifications.

Reply:
In principle, we could pre-read the JSON, or even peek the JSON just for the names.

But in practice, you will have to still connect manually the channel names to the qubit names. The qubit names repetition is the link between the channels definitions and the natives, despite living in two separate files.

Reply:
I would say that link-by-name is now the new standard in 0.2 (instead of the link-by-reference style in 0.1 - which seems clever, but it has been proven plenty of drawbacks).

So, let's say that this is a featur.